src/Microsoft.AspNetCore.Authentication/PersistentAuthenticationHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Antiforgery;
+using Microsoft.AspNetCore.Builder;
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    public abstract class PersistentAuthenticationHandler<TOptions> : AuthenticationHandler<TOptions>
```

Comment:
@blowdart wants it to be easy to opt-in to this behavior in a standard way.  As I mentioned elsewhere, I don't really feel like having a base class in this case helps. It's still easy to get wrong.

I'd either put this all on `AuthenticationHandler` or all on the `Cookie...Handler`